### PR TITLE
html script tag

### PR DESCRIPTION
**Pull request by kbao24**

### CODE REVIEW

File: prep-html-script-tag/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Script Tag</title>
+  </head>
+  <body>
+    <script>
+      console.log('I have added JS to the script tag in the HTML!');
+    </script>
+    <script src="main.js">
+      console.log("I have added a script file via the src attribute!")
```
Comment (BrettAlbright) [inline prep-html-script-tag/index.html]: this can be removed, you only need the src attribute pointing to the main.js file

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update to the code is needed. And make sure to attach a screenshot of the console output as a comment!

**Comment by kbao24** [comment]
html-script-tag

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**